Doc/includes/shoddy.c

```diff
@@ -3,88 +3,52 @@
-    {"increment", (PyCFunction)Shoddy_increment, METH_NOARGS,
+    {"increment", (PyCFunction) Shoddy_increment, METH_NOARGS,
      PyDoc_STR("increment state counter")},
     {NULL},
 };
 
 static int
-Shoddy_init(Shoddy *self, PyObject *args, PyObject *kwds)
+Shoddy_init(ShoddyObject *self, PyObject *args, PyObject *kwds)
 {
-    if (PyList_Type.tp_init((PyObject *)self, args, kwds) < 0)
+    if (PyList_Type.tp_init((PyObject *) self, args, kwds) < 0)
         return -1;
     self->state = 0;
     return 0;
 }
 
-
 static PyTypeObject ShoddyType = {
     PyVarObject_HEAD_INIT(NULL, 0)
-    "shoddy.Shoddy",         /* tp_name */
-    sizeof(Shoddy),          /* tp_basicsize */
-    0,                       /* tp_itemsize */
-    0,                       /* tp_dealloc */
-    0,                       /* tp_print */
-    0,                       /* tp_getattr */
-    0,                       /* tp_setattr */
-    0,                       /* tp_reserved */
-    0,                       /* tp_repr */
-    0,                       /* tp_as_number */
-    0,                       /* tp_as_sequence */
-    0,                       /* tp_as_mapping */
-    0,                       /* tp_hash */
-    0,                       /* tp_call */
-    0,                       /* tp_str */
-    0,                       /* tp_getattro */
-    0,                       /* tp_setattro */
-    0,                       /* tp_as_buffer */
-    Py_TPFLAGS_DEFAULT |
-        Py_TPFLAGS_BASETYPE, /* tp_flags */
-    0,                       /* tp_doc */
-    0,                       /* tp_traverse */
-    0,                       /* tp_clear */
-    0,                       /* tp_richcompare */
-    0,                       /* tp_weaklistoffset */
-    0,                       /* tp_iter */
-    0,                       /* tp_iternext */
-    Shoddy_methods,          /* tp_methods */
-    0,                       /* tp_members */
-    0,                       /* tp_getset */
-    0,                       /* tp_base */
-    0,                       /* tp_dict */
-    0,                       /* tp_descr_get */
-    0,                       /* tp_descr_set */
-    0,                       /* tp_dictoffset */
-    (initproc)Shoddy_init,   /* tp_init */
-    0,                       /* tp_alloc */
-    0,                       /* tp_new */
+    .tp_name = "shoddy.Shoddy",
+    .tp_doc = "Shoddy objects",
+    .tp_basicsize = sizeof(ShoddyObject),
+    .tp_itemsize = 0,
+    .tp_flags = Py_TPFLAGS_DEFAULT | Py_TPFLAGS_BASETYPE,
+    .tp_init = (initproc) Shoddy_init,
+    .tp_methods = Shoddy_methods,
 };
 
 static PyModuleDef shoddymodule = {
     PyModuleDef_HEAD_INIT,
-    "shoddy",
-    "Shoddy module",
-    -1,
-    NULL, NULL, NULL, NULL, NULL
+    .m_name = "shoddy",
+    .m_doc = "Example module that creates an extension type.",
+    .m_size = -1,
 };
 
 PyMODINIT_FUNC
 PyInit_shoddy(void)
 {
-    PyObject *m;
-
+    PyObject* m;
```

Comment:
It is more common to add a space before `*` than after it.

Reply:
Uh, right. I'm confused between the coding styles of two different projects :-(